frame/nomination-pools/src/lib.rs

```diff
@@ -2196,6 +2503,103 @@ pub mod pallet {
 			let signer = ensure_signed(origin)?;
 			Self::do_claim_payout(signer, other)
 		}
+
+		/// Set the commission of a pool.
```

Comment:
woudl be good to mention that existing pending rewards are stored one last time with the old amount.